.travis.yml

```diff
@@ -1,5 +1,11 @@
-language: python
-sudo: false
+language: node_js
+dist: trusty
+
+node_js:
+- "0.10"
+- "4"
+
+sudo: required
```

Comment:
This is a requirement of the new environment, see at the top of the table in the provided link.